website/source/docs/platform/k8s/helm.html.md

```diff
@@ -22,6 +22,14 @@ properly installed and configured with your Kubernetes cluster.
 may still change significantly over time. Please always run Helm with
 `--dry-run` before any install or upgrade to verify changes.
 
+~> **Warning:** By default, the chart will install an insecure configuration
```

Comment:
nit: we often bold the words "Security" or "Security Warning" in messages like this to make it clear. E.g. 

![image](https://user-images.githubusercontent.com/120915/51130858-f5e82880-1825-11e9-8a88-a749cd4f4333.png)

I'd maybe consider `**Security Warning:**` just to make it clear the scope etc.

Reply:
Updated!